WordPress/Classes/Services/BlogJetpackSettingsService.swift

```diff
@@ -0,0 +1,194 @@
+import Foundation
+import CocoaLumberjack
+import WordPressKit
+
+struct BlogJetpackSettingsService {
+
+    fileprivate let context: NSManagedObjectContext
+
+    init(managedObjectContext context: NSManagedObjectContext) {
+        self.context = context
+    }
+
+    /// Sync ALL the Jetpack settings for a blog
+    ///
+    func syncJetpackSettingsForBlog(_ blog: Blog, success: @escaping () -> Void, failure: @escaping (Error?) -> Void) {
+        guard blog.supports(.jetpackSettings) else {
+            success()
+            return
+        }
+        guard let blogInContext = self.context.object(with: blog.objectID) as? Blog,
```

Comment:
If we're now assuming the right context, we can skip the `blogInContext` fetch and use `blog` directly

Reply:
oh yep, forgot to get rid of that one.